Source/Relativity.Test.Helpers/TestHelpersKepler/TestHelpersKepler.Services/TestHelpersModule/v1/TestHelpersService.cs

```diff
@@ -56,6 +56,79 @@ public async Task<GetGuidResponseModel> GetGuidAsync(int artifactID, int workspa
 			}
 		}
 
+		public async Task<FieldArtifactIdResponseModel> GetFieldArtifactIdAsync(string fieldName, int workspaceId)
+		{
+			FieldArtifactIdResponseModel responseModel;
+
+			try
+			{
+				int fieldId = await _helper.GetDBContext(workspaceId).ExecuteScalarAsync<int>(
+					new ContextQuery
+					{
+						SqlStatement = "SELECT [ArtifactID] FROM [EDDSDBO].[Field] WHERE [DisplayName] LIKE @fieldName",
```

Comment:
i think we should use ExtendedField view instead of Field table for queries as it will filter out an fields which are marked for deletion.